### ISSUE TITLE
Firebase user Authentication

### ISSUE THREAD

**Issue by kourilm83** [Get-Me-Fit/getmefit-mobile#2]
**Given** I am first time user of GMF
**and** I have the app installed
**When** I decide to login
**Then** I can log in with email and password
**or** I can log in with Facebook
**or** I can log in with Google

For MVP we will require the user to login every time they start the app, later we will make it more UX friendly.

Useful links:
https://medium.com/firebase-developers/firebase-auth-on-xamarin-forms-171432aa3d76

**Comment by kourilm83** [comment]
I finished email and password sign-in/up, for now, will switch to another task to unblock other issues

**Comment by kourilm83** [comment]
Now working on Google sign in
